src/main/webapp/resources/js/apis/samples/samples.js

```diff
@@ -1,7 +1,25 @@
 import axios from "axios";
 import { setBaseUrl } from "../../utilities/url-utilities";
+import { notification } from "antd";
 
-const URL = setBaseUrl(`samples`);
+const URL = setBaseUrl(`ajax/samples`);
 
-export const getSampleDetails = id =>
-  axios.get(`${URL}?id=${id}`).then(response => response.data);
+export const fetchSampleDetails = async (id) => {
+  try {
+    const { data } = await axios.get(`${URL}/${id}/details`);
+    return data;
+  } catch (e) {
+    return Promise.reject(e.response.data.error);
+  }
+};
+
+export async function fetchSampleFiles({ sampleId, projectId }) {
+  try {
+    const response = await axios(
+      `${URL}/${sampleId}/files${projectId ? `?projectId=${projectId}` : null}`
+    );
+    return response.data;
+  } catch (e) {
+    notification.error({ message: e.response.data.message });
```

Comment:
Why are we displaying a `notification.error here`? I think it would be better to be in the UI file where this is called. We should leave this to just make the api call and return the data or error

Reply:
I have no clue, let me fix that.

Reply:
Updated in 94bab472469c60e4cd4d889d7d5bc69a1cb7b1af